packages/common/validation.js

```diff
@@ -28,3 +32,28 @@ export const alphaNumeric = value =>
 // Phone number validation
 export const phoneNumber = value =>
   value && !/^(0|[1-9][0-9]{9})$/i.test(value) ? 'Invalid phone number, must be 10 digits' : undefined;
+
+export const validateForm = (fValues, fSchema) => {
```

Comment:
why the naming `fValues`, `fSchema` here? should be `formValues`, `formSchema`

Reply:
Fixed